### PR TITLE
RSSGuard: new recipe

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

**Comment by extrowerk** [comment]
Need to add an icon and adjust the rdef

**Comment by extrowerk** [comment]
Please help me create an icon for this.

**Comment by diversys** [comment]
Cast @threedeyes

**Comment by korli** [comment]
Could you consider updating to https://github.com/martinrotter/rssguard/releases/tag/4.0.2 then merge this PR?

File: net-news/rssguard/rssguard-4.0.2.recipe [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+* FreshRSS,
+* The Old Reader,
+* Bazqux,
+* Reedah,
+* Feedly."
+HOMEPAGE="https://github.com/martinrotter/rssguard"
+COPYRIGHT="2011-2021 Martin Rotter"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/martinrotter/rssguard/archive/refs/tags/4.0.2.tar.gz"
```
Comment (korli) [inline net-news/rssguard/rssguard-4.0.2.recipe:18]:
```suggestion
SOURCE_URI="https://github.com/martinrotter/rssguard/archive/refs/tags/$portVersion.tar.gz"
```

Comment (extrowerk) [reply net-news/rssguard/rssguard-4.0.2.recipe]: done

**Review by korli — APPROVED** [review]

**Pull request merged**